WordPress/Classes/ViewRelated/Blog/EditSiteViewController.m

```diff
@@ -138,46 +162,41 @@ - (void)viewDidAppear:(BOOL)animated
 
 #pragma mark - UITableView
 
-- (NSInteger)numberOfSectionsInTableView:(UITableView *)tv
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
 {
     if (self.blog && ![self.blog isWPcom]) {
-        return 3;
+        return EditSiteSectionCountSelfHosted;
     }
-    return 2;
+    return EditSiteSectionCount;
 }
 
-- (NSInteger)tableView:(UITableView *)tv numberOfRowsInSection:(NSInteger)section
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
 {
-    switch (section) {
-        case 0:
-            // URL, username, [password]
-            if ([self.blog isWPcom]) {
-                return 2;
-            }
-
-            return 3;
-        case 1:
-            // Settings: Geolocation, [ Push Notifications ]
-            if ([self canTogglePushNotifications]) {
-                return 2;
-            }
-
-            return 1;
-        case 2:
-            return 1;
-        default:
-            break;
+    if (section == TableViewSectionTitle) {
+        if ([self.blog isWPcom]) {
+            return EditSiteRowCountForSectionTitle;
+        }
+        return EditSiteRowCountForSectionTitleSelfHosted;
+    } else if (section == TableViewSectionSettings) {
+        if ([self.blog isWPcom]) {
```

Comment:
Already fixed in #3725. I got a conflict when merging and I changed it back, just wanted to make sure I wasn't breaking anything